### PR TITLE
Update towncrier.toml

### PR DESCRIPTION
**Pull request by ClaireGizard**
#319 

Update Tomcrier configuration.
When you want to create a new version, you have to update the "NEW_VERSION" in the towncrier.toml file with current version name

Capture

To update the TCHAP_CHANGES.md
Install Python and towncrier: https://github.com/twisted/towncrier
run: " towncrier build"

### CODE REVIEW

File: changelog.d/307.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Application tabs are not updated correctly #307
```
Comment (Florian14) [inline changelog.d/307.bugfix:1]: Reference to the issue is not necessary anymore as it is already in the filename and will be automatically added by towncrier

File: towncrier.toml [hunk 2]
```diff
@@ -1,9 +1,10 @@
 [tool.towncrier]
+    version = "1.99.4"
```
Comment (Florian14) [inline towncrier.toml:2]: In Element, version was not declared here, do you know why ? Maybe it is only set locally when running towncrier and not pushed to the repository ?

File: towncrier.toml [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
     directory = "changelog.d"
-    filename = "CHANGES.md"
-    name = "Changes in Element"
+    filename = "TCHAP_CHANGES.md"
+    name = "Changes in Tchap"
     template = "tools/towncrier/template.md"
-    issue_format = "[#{issue}](https://github.com/vector-im/element-android/issues/{issue})"
+    issue_format = "[#{issue}](https://github.com/tchapgouv/tchap-android-v2/issues)"
```
Comment (Florian14) [inline towncrier.toml:7]: please keep `/{issue}` at the end of the url

File: towncrier.toml (hunk 2 above)
Comment (ClaireGizard) [reply towncrier.toml:2]: i think they add it each time they create a new version.

**Review by Florian14 — APPROVED** [review]
LGTM

**Pull request merged**